doc/sphinx-guides/source/admin/external-tools.rst

```diff
@@ -94,6 +94,14 @@ File level preview tools allow the user to see a preview of the file contents wi
 
 When a file has a preview available, a preview icon will appear next to that file in the file listing on the dataset page. On the file page itself, the preview will appear in a Preview tab either immediately or once a guestbook has been filled in or terms, if any, have been agreed to.
 
+File Level Query Tools
+++++++++++++++++++++++++
+
+File level query tools allow the user to ask natural language queries of data table's contents without having to download it.
```

Comment:
```suggestion
File level query tools allow the user to ask questions (e.g. natural language queries) of a data table's contents without having to download it.
```

Maybe this should be a little less specific to Ask Dataverse. Please see a suggested change above.